src/snapred/backend/data/LocalDataService.py

```diff
@@ -82,37 +89,37 @@ def _createFileNotFoundError(msg, filename):
 
 @Singleton
 class LocalDataService:
-    instrumentConfig: "InstrumentConfig"
-    verifyPaths: bool = True
-
     # conversion factor from microsecond/Angstrom to meters
     # (TODO: FIX THIS COMMENT! Obviously `m2cm` doesn't convert from 1.0 / Angstrom to 1.0 / meters.)
     CONVERSION_FACTOR = Config["constants.m2cm"] * PhysicalConstants.h / PhysicalConstants.NeutronMass
 
     def __init__(self) -> None:
-        self.verifyPaths = Config["localdataservice.config.verifypaths"]
-        self.instrumentConfig = self.readInstrumentConfig()
+        self._verifyPaths = Config["localdataservice.config.verifypaths"]
+        self._instrumentConfig = self._readInstrumentConfig()
         self.mantidSnapper = MantidSnapper(None, "Utensils")
 
     ##### MISCELLANEOUS METHODS #####
 
+    def getInstrumentConfig(self):
+        return self._instrumentConfig
+
     def fileExists(self, path):
         return os.path.isfile(path)
 
     def _determineInstrConfigPaths(self) -> None:
         """This method locates the instrument configuration path and
-        sets the instance variable ``instrumentConfigPath``."""
+        sets the instance variable ``_instrumentConfigPath``."""
```

Comment:
too bad this stuff is going away in the historical SNAPInstPrm pr I'm working on!